### PR TITLE
Update installation.rst to reflect changes due to swig

### PR DESCRIPTION
**Pull request by soumenganguly**
Currently, M2Crypto is only compatible with swig <=3.0.4 and thus upgraded versions cause ``make full_init`` to fail. This Patch mention's this exception in the installation.rst file, so that users can account and react to the exception accordingly.

### CODE REVIEW

File: docs/topics/install-olympia/installation.rst [hunk 1]
```diff
@@ -36,6 +36,8 @@ if you're running a recent version, you can `install them automatically
 
     sudo aptitude install python-dev python-virtualenv libxml2-dev libxslt1-dev libmysqlclient-dev memcached libssl-dev swig openssl curl libjpeg-dev zlib1g-dev libsasl2-dev
 
+**Note**:
```
Comment (magopian) [inline docs/topics/install-olympia/installation.rst]: You could use the same syntax (`.. note::`) as below in the file, on line 204, what do you think?

**Comment by soumenganguly** [comment]
Made the changes in the recent commit, let me know if it needs some changes @magopian

**Comment by magopian** [comment]
This looks perfect, thanks!

**Comment by magopian** [comment]
Merged in https://github.com/mozilla/olympia/commit/0e16225089f1621b3adee48f5bd9bb926c22953a

**Pull request closed without merging**